docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -171,6 +171,124 @@ This triggers controllers watching the object to execute any finalizers they han
 Each controller then removes its finalizer from the list and issues the delete request again.
 This request only deletes the object if the list of finalizers is now empty,
 meaning all finalizers are done.
+
+### Validation
+
+Validation of custom objects is possible via [OpenAPI v3 schema](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schemaObject).
+Additionally, the following restrictions are applied to the schema:
+
+- The fields `default`, `nullable`, `discriminator`, `readOnly`, `writeOnly`, `xml` and
```

Comment:
not support = rejected?

Reply:
or ignored?

Reply:
rejected.
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L298

Reply:
then we should better write "cannot be set".

Reply:
Done.